src/kernels/fbm.jl

```diff
@@ -0,0 +1,51 @@
+"""
+    FBMKernel(; h::Real=0.5)
+Fractional Brownian motion kernel with Hurst index h from (0,1) given by
+```
+    κ(x,y) =  ( |x|²ʰ + |y|²ʰ - |x-y|²ʰ ) / 2
+```
+
+For h=1/2, this is the Wiener Kernel, for h>1/2, the increments are
+positively correlated and for h<1/2 the increments are negatively correlated.
+%
+"""
+struct FBMKernel{T<:Real} <: BaseKernel
+    h::T
+    function FBMKernel(;h::T=0.5) where {T<:Real}
+        @assert h<=1.0 && h>=0.0 "FBMKernel: Given Hurst index h is invalid."
+        new{T}(h)
+    end
+end
+
+_fbm(modX, modY, modXY, h) = (modX^h + modY^h - modXY^h)/2
+
+function kernelmatrix(κ::FBMKernel, X::AbstractMatrix; obsdim::Int = defaultobs)
+    @assert obsdim ∈ [1,2] "obsdim should be 1 or 2 (see docs of kernelmatrix))"
+    K = map(modX->_fbm(modX, modX, 0, κ.h), pairwise(SqEuclidean(),X,dims=obsdim))
```

Comment:
Shouldn't this rather be
```suggestion
    modX = sum(abs2, X; dims = 3 - obsdim)
    modXX = pairwise(SqEuclidean(), X, dims = obsdim)
    K = _fbm.(vec(modX), reshape(modX, 1, :), modXX, κ.h)
```
?